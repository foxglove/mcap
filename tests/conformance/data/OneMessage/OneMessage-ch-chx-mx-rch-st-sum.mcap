version https://git-lfs.github.com/spec/v1
oid sha256:19af0427911a999307b21ea9b935e2d5dfad6e2ef3a0fc5a6fd1146dea7681e8
size 543
