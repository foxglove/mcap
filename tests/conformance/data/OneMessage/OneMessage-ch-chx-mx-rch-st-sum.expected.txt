Header library="" profile=""
Schema id=1 schema=b schema_encoding=c schema_name=Example
ChannelInfo id=1 message_encoding=a metadata={foo=bar} schema_id=1 topic=example
Message channel_id=1 log_time=2 message_data=<010203> publish_time=1 sequence=10
MessageIndex channel_id=1 records={2=79}
ChannelInfo id=1 message_encoding=a metadata={foo=bar} schema_id=1 topic=example
Statistics attachment_count=0 channel_count=1 channel_message_counts={1=1} chunk_count=1 message_count=1
ChunkIndex chunk_length=160 chunk_start_offset=25 compressed_size=113 compression="" end_time=2 message_index_length=31 message_index_offsets={1=187} start_time=2 uncompressed_size=113
SummaryOffset group_length=47 group_opcode=4 group_start=231
SummaryOffset group_length=0 group_opcode=13 group_start=321
SummaryOffset group_length=43 group_opcode=11 group_start=278
SummaryOffset group_length=0 group_opcode=10 group_start=321
SummaryOffset group_length=87 group_opcode=8 group_start=321
Footer summary_crc=0 summary_offset_start=408 summary_start=231
